apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -367,7 +371,7 @@ NetSimPacketEditor.prototype.makeKeyupHandler = function (fieldName,
     converterFunction, fieldWidth) {
   return function (jqueryEvent) {
     var newValue = converterFunction(jqueryEvent.target.value, fieldWidth);
-    if (!isNaN(newValue)) {
+    if (typeof newValue === 'string' || !isNaN(newValue)) {
       this[fieldName] = newValue;
       this.updateFields_(jqueryEvent.target);
```

Comment:
Fun with NaN:

- Converting 0001 to "1": Totally fine, because `isNaN("1") // false` because it tries to convert the string
- Likewise 0001 0001 to "1.1": `isNaN("1.1") // false`
- But at three-part, 0000 0000 0001 to "1.1.1": `isNaN("1.1.1") // true`

Basically, anything that converts to a string is fine by me here, the NaN check is for conversions to numbers and not wanting to stick "NaN" into the UI.